shared/stories/storybook.shared.js

```diff
@@ -24,9 +40,21 @@ let uniqueProviderKey = 1
 const createPropProvider = (...maps: SelectorMap[]) => {
   const merged: SelectorMap = maps.reduce((obj, merged) => ({...obj, ...merged}), {})
 
+  /*
+   * GatewayDest and GatewayProvider need to be wrapped by the Provider here in
+   * order for storybook to correctly mock connected components inside of
+   * popups.
+   * React.Fragment is used to render StorybookErrorBoundary and GatewayDest as
+   * children to GatewayProvider which only takes one child
+   */
   return (story: () => React.Node) => (
     <Provider key={`provider:${uniqueProviderKey++}`} store={createStore(state => state, merged)}>
-      <StorybookErrorBoundary children={story()} />
+      <GatewayProvider key="gatewayprovider">
+        <React.Fragment>
+          <StorybookErrorBoundary children={story()} />
+          <GatewayDest component={DestBox} name="popup-root" />
```

Comment:
@chrisnojima Danny mentioned that `GatewayDest` won't render for stories have popups but don't use `createPropProvider`.
We discussed it briefly and concluded that using `createPropProvider` is a reasonable requirement for rendering popups. Added a comment to `FloatingMenu` at his suggestion.